doc/tutorials/validation.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction ##
 
-Configuration in FLOSS unfortunately is often stored
+Configuration in <abbr title="Free Linux Open Source Software">FLOSS</abbr> unfortunately is often stored
```

Comment:
It is "Free/Libre and Open Source Software", see https://www.gnu.org/philosophy/floss-and-foss.en.html